homeassistant/components/climate/mill.py

```diff
@@ -194,3 +205,14 @@ def max_temp(self):
     async def async_update(self):
         """Retrieve latest state."""
         self._heater = await self._conn.update_device(self._heater.device_id)
+
+    async def async_set_operation_mode(self, operation_mode):
+        """Set operation mode."""
+        if operation_mode == STATE_HEAT:
+            await self.async_turn_on()
+        elif operation_mode == STATE_OFF:
+            await self.async_turn_off()
+        else:
+            _LOGGER.error("Unrecognized operation mode: %s", operation_mode)
+            return
+        self.schedule_update_ha_state()
```

Comment:
Change to async version. We're in async land here.

Reply:
Hold on, we shouldn't schedul a state update at all. This is a polling entity so the service call will schedule a state update.

Reply:
good catch

Reply:
I think that is not needed because it's a polling device?

Reply:
Exactly!

Reply:
Ahh, sorry.
Will fix it